src/main/scala/mesosphere/marathon/api/akkahttp/v2/PodsController.scala

```diff
@@ -100,7 +100,36 @@ class PodsController(
 
   def find(podId: PathId): Route = ???
 
-  def remove(podId: PathId): Route = ???
+  def remove(podId: PathId): Route =
+    authenticated.apply { implicit identity =>
+      (extractClientIP & forceParameter) { (clientIp, force) =>
+        extractRequest { req =>
```

Comment:
Can we extract URI instead of request here?

Reply:
Sure. Much better. Thanks.